flow_action_components/GetRecordTypeInfobyObject/README.md

```diff
@@ -0,0 +1,66 @@
+Get Record Type Info by Object - Flow Action
+
+This action can be used to generate a list of record types for a specific object. This can be useful when you would like to present a record type selector to the user. Can be used as inputs to the QuickChoice managed package by setting Input Mode to “Dual String Collections”. Pass in one output collection for Labels and one for the underlying values.
+
+More info here: https://unofficialsf.com/get-record-type-info-by-object---flow-action/
```

Comment:
Really good work, Tamar. I'll go ahead and merge. Note the following very minor documentation items:

1) In your post, use 'true' instead of 'on'. Because you're dealing with Booleans, everything should always be true or false.

2) In your post, if you're going to document a string and the structure of the string is important, make sure to mention that. Here, the record type Id's are separated by commas. That kind of detail is super important:

recordTypeIds | String | Ids of the records types.
-

3) As part of posting, create a row in the Flow Actions table at https://unofficialsf.com/flow-actions/ . To get there, go into the Wordpress setup and find TablePress.